### PR TITLE
Update to SUSHI v1.2.0

### LINKED ISSUE #34: Broken on Safari
Oof.  FSH Online doesn't work _at all_ in Safari.  Just a white screen.
image

And here's the console:
image

### PR DESCRIPTION
**Pull request by jafeltra**
This PR updates the installed SUSHI to the latest version, v1.2.0. This SUSHI update has a fix for using regex lookbehind assertions, and it fixes #34 so the app will load in Safari.

### CODE REVIEW

**Comment by jafeltra** [comment]
dfc3be6 also updates the helper text under the "Dependencies" field to include US Core as an example and change the wording to `packageId`, based on feedback from @markkramerus. This was such a small change I just decided to lump it in here.

File: src/components/SUSHIControls.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ export default function SUSHIControls(props) {
               margin="dense"
               fullWidth
               label="Dependencies"
-              helperText="dependencyID#version, dependencyID#version"
+              helperText="Format: packageId#version, packageId#version (ex. hl7.fhir.us.core#3.1.1)"
```
Comment (cmoesel) [inline src/components/SUSHIControls.js:218]: This is a nit, but isn't it customary to use `e.g.,` in a situation like this rather than `ex.`?

Comment (jafeltra) [reply src/components/SUSHIControls.js:218]: I almost used that, so I'm happy to swap it out!

Comment (jafeltra) [reply src/components/SUSHIControls.js]: 21b4522 swapped!

**Review by ngfreiter — APPROVED** [review]
This looks good, and I tested features from 1.2.0, and they seem to be working.

**Pull request merged**